FirebaseDev.podspec

```diff
@@ -16,6 +16,7 @@ Simplify your iOS development, grow your user base, and monetize more effectivel
   s.source           = { :git => 'https://github.com/firebase/firebase-ios-sdk.git', :tag => s.version.to_s }
   s.social_media_url = 'https://twitter.com/Firebase'
   s.ios.deployment_target = '8.0'
+  s.osx.deployment_target = '10.10'
```

Comment:
Since we're not yet building and publishing the iOS CocoaPod from the open source repo, it might also be useful to make a separate MacOS podspec in the interim.